pokemongo_bot/cell_workers/find_biggest_cluster.py

```diff
@@ -0,0 +1,40 @@
+from pokemongo_bot.cell_workers.utils import distance, merc2coord, coord2merc
+from networkx.algorithms.clique import find_cliques
+
+import networkx as nx
+import numpy as np
+
+
+class FindBiggestCluster(object):
+    def __init__(self, bot, config):
+        self.bot = bot
+        self.dest = None
+        self.config = config
+        self._process_config()
+
+    def _process_config(self):
+        self.radius = self.config.get("radius", 50)
+
+    def work(self):
+        forts = self.bot.get_forts()
+        self.dest = find_biggest_cluster(self.radius, forts)
+
+
+def find_biggest_cluster(radius, points):
```

Comment:
This isn't required, we don't have tests for many other things, but would you be willing to write a few unit tests for this? It seems like something that would benefit greatly from having a few test cases proving that it works and that we don't break it in future PRs. We have some tests for some other classes already in the tests folder.